templates/repo/issue/view_title.tmpl

```diff
@@ -1,19 +1,24 @@
-<div class="sixteen wide column title">
+<div class="issue-title-header">
 	<div class="issue-title" id="issue-title-wrapper">
-		{{if and (or .HasIssuesOrPullsWritePermission .IsIssuePoster) (not .Repository.IsArchived)}}
-			<button id="edit-title" class="ui basic button secondary edit-button not-in-edit">{{.locale.Tr "repo.issues.edit"}}</button>
-		{{end}}
-		<h1>
-			<span id="issue-title">{{RenderIssueTitle $.Context .Issue.Title $.RepoLink $.Repository.ComposeMetas | RenderCodeBlock}}</span>
-			<span class="index">#{{.Issue.Index}}</span>
-			<div id="edit-title-input" class="ui input gt-ml-4 gt-hidden">
+		<h1 class="gt-df gt-ac gt-f1 gt-word-break">
```

Comment:
No idea why these styles are not put in `.issue-title h1`, which already has many fine tunes styles like margin/padding

```suggestion
		<h1 class="gt-df gt-ac gt-f1 gt-word-break">
```
![image](https://user-images.githubusercontent.com/2114189/235663034-bb89d14f-d777-4a82-9975-fa70be55e88e.png)

Reply:
At least `gt-word-break` should stay for re-use.

Reply:
`gt-word-break` seems good

Reply:
moved styles